stingray/powerspectrum.py

```diff
@@ -114,17 +116,19 @@ def __init__(self, data=None, norm="frac", gti=None,
                 dt=dt
             )
 
+        self.norm = norm.lower()
+        self.dt = dt
+
         if not good_input:
             self.freq = None
             self.power = None
             self.power_err = None
             self.df = None
             self.dt = None
+            self.nphots = None
             self.nphots1 = None
```

Comment:
Do we need `nphots` and `nphots1` here? Or can we get away with just `nphots`?

Reply:
It's for compatibility with Crossspectrum. It needs to be there as long as we have the legacy interface working